server/middleware/setUpSentry.ts

```diff
@@ -0,0 +1,55 @@
+import * as Sentry from '@sentry/node'
+import express from 'express'
+import applicationVersion from '../applicationVersion'
+import config from '../config'
+
+export function setUpSentryRequestHandler(app: express.Express): void {
+  if (config.sentry.dsn) {
+    // Prevent usernames which are PII from being sent to Sentry
+    // https://docs.sentry.io/platforms/python/guides/logging/data-management/sensitive-data/#examples
+    const anonymousId = Math.random().toString()
+    Sentry.setUser({ id: anonymousId, username: anonymousId })
```

Comment:
Unfortunately this didn't prevent the username being sent as a "tag" for us - I've merged another PR that appears to have fixed it [here](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/99) so on the off chance this doesn't work for you either, you may want to follow our example

Reply:
argh, good catch, it is sharing the local username 😱